about.html

```diff
@@ -12,62 +13,114 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <img src="img/lambda-black.png" alt="black lambda logo">
+        </div>
+        <div class="links">
+          <a href="index.html">Home</a>
+          <a href="#">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+      </nav>
+    </header>
 
+    <section class="jumbo-img">
+      <!--empty-->
+    </section>
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <div class="container-about-page">
+      <section class="about-content">
+        <h2>About</h2>
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to<br>
+          corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow
+          the<br> holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </div>
+    </section>
 
-    Learn More
 
```

Comment:
Good job using semantic elements to describe the layout and reusing classes for the same styling.